images/saige-qtl/Dockerfile

```diff
@@ -20,11 +20,25 @@ RUN apt-get update && \
     R CMD INSTALL . && \
     mv extdata/step1_fitNULLGLMM_qtl.R \
        extdata/step2_tests_qtl.R \
-       extdata/step3_LDmat.R \
-       extdata/createSparseGRM.R /usr/local/bin/ && \
+       extdata/step3_gene_pvalue_qtl.R \
+       extdata/input/n.indep_100_n.cell_1_01.step1 \
+       extdata/input/seed_1_100_nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_Poisson.txt \
+       extdata/input/genotype_10markers.vcf.gz \
+       extdata/input/genotype_10markers.vcf.gz.csi \
+       extdata/input/gene_1_cis_region.txt \
+       extdata/output/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1 \
+       extdata/output/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis \
+       extdata/output/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis_genePval /usr/local/bin/ && \
     chmod a+x /usr/local/bin/step1_fitNULLGLMM_qtl.R \
               /usr/local/bin/step2_tests_qtl.R \
-              /usr/local/bin/step3_LDmat.R \
-              /usr/local/bin/createSparseGRM.R && \
+              /usr/local/bin/step3_gene_pvalue_qtl.R \
+              /usr/local/bin/n.indep_100_n.cell_1_01.step1 \
+              /usr/local/bin/seed_1_100_nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_Poisson.txt \
+              /usr/local/bin/genotype_10markers.vcf.gz \
+              /usr/local/bin/genotype_10markers.vcf.gz.csi \
+              /usr/local/bin/gene_1_cis_region.txt \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1 \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis_genePval && \
```

Comment:
I don't think the data files here need to be made executable 🤔

Reply:
haha probably not no they are just txt files for the most part, and a vcf! So I just need to do the move bit, not the chmod?

Reply:
Yeah, the chmod will be required for code files that you intend to execute directly (e.g. `$ /usr/local/bin/step3_gene_pvalue_qtl.R <args>`), if you run them using RSCRIPT you may not need to make them executable at all, but if it's worked so far maybe keep it as-is